crates/air-parser/src/parser/lexer/errors.rs

```diff
@@ -32,6 +35,20 @@ pub enum LexerError {
 
     #[error("invalid character in json path")]
     InvalidJsonPath(usize, usize),
+
+    #[error("this signed digit could have only digits or one dot")]
+    UnallowedCharInNumber(usize, usize),
+
+    #[error("{2}")]
```

Comment:
could 
```
    #[error(transparent)]
    ParseIntError(usize, usize, #[from] anyhow::Error), 
```
be used here?

Reply:
Afaik `#[error(transparent)]` requires exactly one field in a enum variant.